web build error

I get this error when running "npm run web"

```
The CLI moved into a separate package: webpack-cli.
Please install 'webpack-cli' in addition to webpack itself to use the CLI.
-> When using npm: npm install webpack-cli -D
-> When using yarn: yarn add webpack-cli -D
module.js:557
    throw err;
    ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\hjtho\Google Drive\Development\Web\cunt\node_modules\webpack-dev-server\bin\webpack-dev-server.js:65:1)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! rnsk@0.1.0 web: `webpack-dev-server --content-base build/ --config webpack/web.dev.config.js --port 3001 --inline --hot --colors --historyApiFallback`
npm ERR! Exit status 1
```

Windows 10, latest master version.

I've tried all sorts of things like having this webpack-cli but then I get a different error about how 'loaders' isn't recognized in the web.config.js file

--- a/package.json
+++ b/package.json
@@ -26,8 +26,8 @@
     "redux-logger": "^3.0.6",
     "sass-loader": "^7.0.1",
     "style-loader": "^0.21.0",
-    "webpack": "^4.6.0",
-    "webpack-dev-server": "^3.1.3",
+    "webpack": "^3.11.0",
+    "webpack-dev-server": "^2.11.2",
     "webpack-hot-middleware": "^2.22.1"
   },
   "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",

